lib/websocket-tracker.js

```diff
@@ -32,7 +32,10 @@ function WebSocketTracker (client, announceUrl, opts) {
   self._interval = null
 
   if (socketPool[announceUrl]) self._socket = socketPool[announceUrl]
-  else self._socket = socketPool[announceUrl] = new Socket(announceUrl)
+  else {
+      self._socket = socketPool[announceUrl] = new Socket(announceUrl)
+      self._socket.on('ready', function () { self._socket.ready = true })
```

Comment:
Ah, found the issue: https://github.com/feross/bittorrent-tracker/blob/master/package.json#L36. I'll change that instead and fix up the failing build issues.